R/zzz.R

```diff
@@ -1,3 +1,5 @@
+#' @importFrom usethis edit_file
```

Comment:
Interesting. I've not seen folks moving dependencies away from the functions they are called in. To me that has the potential of accidentally removing a dependency (import) when you think you no longer need it and cluttering up another location 🤔